src/oc_erchef/apps/oc_chef_authz/test/oc_chef_authz_acl_tests.erl

```diff
@@ -1,4 +1,4 @@
-%% -*- erlang-indent-level: 4;indent-tabs-mode: nil; fill-column: 92-*-
+%% -*- erlang-indent-level: 3;indent-tabs-mode: nil; fill-column: 92-*-
```

Comment:
vim auto-decremented the number for me due to accidentally hitting a keybinding.  Will fix.